net/crowdsec/Makefile

```diff
@@ -0,0 +1,189 @@
+#
+# Copyright (C) 2021 Gerald Kerma
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=crowdsec
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/crowdsecurity/crowdsec
+PKG_SOURCE_VERSION:=73e0bbaf93070f4a640eb5a22212b5dcf26699de
+PKG_SOURCE_DATE:=20210707
+PKG_MIRROR_HASH:=940e6e122097468ad99fcee2db569afda94fc2a65837c74b0ee71bbb5c64518c
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+
+PKG_BUILD_DEPENDS:=golang/host jq/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+CWD_SYSTEM:=openwrt
+CWD_BUILD_VERSION:=v$(PKG_VERSION)
+CWD_BUILD_GOVERSION:=$(shell go version | cut -d " " -f3 | sed -E 's/[go]+//g')
+CWD_BUILD_CODENAME:=alphaga
+CWD_BUILD_TIMESTAMP:=$(shell date +%F"_"%T)
+CWD_BUILD_TAG:=73e0bbaf93070f4a640eb5a22212b5dcf26699de
+
+CWD_VERSION_PKG:=github.com/crowdsecurity/crowdsec/pkg/cwversion
+
+GO_PKG:=github.com/crowdsecurity/crowdsec
+
+GO_PKG_LDFLAGS_X:=$(CWD_VERSION_PKG).Version=$(CWD_BUILD_VERSION) \
+	$(CWD_VERSION_PKG).System=$(CWD_SYSTEM) \
+	$(CWD_VERSION_PKG).BuildDate=$(CWD_BUILD_TIMESTAMP) \
+	$(CWD_VERSION_PKG).Codename=$(CWD_BUILD_CODENAME)  \
+	$(CWD_VERSION_PKG).Tag=$(CWD_BUILD_TAG) \
+	$(CWD_VERSION_PKG).GoVersion=$(CWD_BUILD_GOVERSION)
+
+TARGET_LDFLAGS+= -static 
+
+GO_PKG_INSTALL_ALL:=1
+MAKE_PATH:=$(GO_PKG_WORK_DIR_NAME)/build/src/$(GO_PKG)/cmd/
+MAKE_VARS += \
+	GOPATH=$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/cmd/ \
+	GOCACHE=$(GO_BUILD_CACHE_DIR) \
+	GOROOT_FINAL=$(GO_TARGET_ROOT) \
+	CC=$(TARGET_CC) \
+	CXX=$(TARGET_CXX) \
+	$(GO_PKG_VARS)
```

Comment:
Not necessary to set `TARGET_LDFLAGS` or `MAKE_*` variables.

Reply:
removed,
thanks